opac/webapp/controllers.py

```diff
@@ -75,6 +75,21 @@ class PreviousOrNextArticleNotFoundError(Exception):
     ...
 
 
+def now():
+    return datetime.utcnow().isoformat()[:10]
+
+
+def add_filter_without_embargo(kwargs={}):
```

Comment:
Não é necessário fazer...., mas é sempre uma boa prática argumentos padrão quando [] ou {}, ser definido como None e ser avaliado dentro da função ou método. 

Veja nesse link na seção **Best Pratice**: https://www.geeksforgeeks.org/default-arguments-in-python/?ref=lbp